src/Blocks/components/checkboxes/checkboxes.php

```diff
@@ -42,27 +44,64 @@
 // Additional content filter.
 $additionalContent = UtilsGeneralHelper::getBlockAdditionalContentViaFilter('checkboxes', $attributes);
 
+$placeholderLabel = '';
+$placeholder = '';
+$checkboxesHideLabel = false;
+$checkboxesFieldLabel = $attributes[Components::getAttrKey('checkboxesFieldLabel', $attributes, $manifest)] ?? '';
+$checkboxesShowAs = $attributes[Components::getAttrKey('checkboxesShowAs', $attributes, $manifest)] ?? '';
+
+// Checkboxes don't use placeholder so we are not going to render it.
+if ($checkboxesShowAs !== '') {
+	// Placeholder input value.
+	if ($checkboxesPlaceholder) {
+		$placeholderLabel = $checkboxesPlaceholder;
+	}
+
+	// Placeholder label for value.
+	if ($checkboxesUseLabelAsPlaceholder) {
+		$checkboxesHideLabel = true;
+		$placeholderLabel = esc_attr($checkboxesFieldLabel) ?: esc_html__('Select option', 'eightshift-forms'); // phpcs:ignore WordPress.PHP.DisallowShortTernary.Found
+	}
+
+	$placeholder = Components::render(
+		'checkbox',
+		[
+			'checkboxLabel' => $placeholderLabel,
+			'checkboxAsPlaceholder' => true,
+			'checkboxIsHidden' => true,
+		]
+	);
+}
+
 $checkboxes = '
+	' . $placeholder . '
 	' . $checkboxesContent . '
 	' . $additionalContent . '
 ';
```

Comment:
```suggestion
$checkboxes = "
    $placeholder
    $checkboxesContent
    $additionalContent
";

```
can't you use just the string interpolation here?